packages/xo-server-load-balancer/src/plan.js

```diff
@@ -314,9 +314,19 @@ export default class Plan {
   async _processVcpuPrepositioning(hosts) {
     const promises = []
 
-    const idToHost = keyBy(hosts, 'id')
+    // removing hosts which have incorrect cpu count value to avoid mass migration on rrd malfunction
+    const sanitizedHostList = hosts.filter(host => parseInt(host.CPUs.cpu_count) > 0)
+    if (sanitizedHostList.length < hosts.length) {
+      const unhealthyHost = hosts.find(host => !(parseInt(host.CPUs.cpu_count) > 0))
+      warn(`vCPU balancing: A host has unexpected CPU value: ${inspect(unhealthyHost.CPUs, { depth: null })}`)
+      if (sanitizedHostList.length < 2) {
+        // need at least 2 hosts
+        return
+      }
```

Comment:
Maybe throw/log an error?
If I understand, it means VM should be migrated, but won't because there aren't enough hosts